server/src/module/student/grading.service.ts

```diff
@@ -158,35 +149,31 @@ export class GradingService {
      * @throws `BadRequestException` - If the students of the team have different gradings.
      */
     async setOfTeam(team: Team, dto: GradingDTO): Promise<void> {
-        const em = this.entityManager.fork({ clear: false });
-        await em.begin();
+        const handIn = await this.getHandInFromDTO(dto);
```

Comment:
Same as above: Should this be wrapped inside a transaction?

Reply:
to my understanding, this works as we use [implicit transactions](https://mikro-orm.io/docs/transactions#approach-1-implicitly), however, if you want to, I can switch to explicit transactions

Reply:
As above, implicit transactions are enough, I think.